src/map/lua/luautils.h

```diff
@@ -30,17 +30,17 @@
 #include "lua.hpp"
 #include "lunar/lunar.h"
 
-/*
-#define SOL_LUA_VERSION 501
-#define SOL_LUAJIT_VERSION 20000
-#define SOL_LUAJIT 1
 #define SOL_ALL_SAFETIES_ON 1
 #include "sol/sol.hpp"
 
+// sol changes this behaviour to return 0 rather than trancate
```

Comment:
Not a game-breaker by any means, but trancate->truncate.  Just being nit-picky 👍